addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -72,6 +74,18 @@
 		<label>Play URI</label>
 		<description>Play the given URI</description>
 	</channel-type>
+    <channel-type id="pvropentv" advanced="true">
```

Comment:
I changed the indentation and commited it in new PR [#2084](https://github.com/openhab/openhab2-addons/pull/2084). Sry for the circumstances.